### PR TITLE
boot: set kernel command line in modeenv during install 

### PR DESCRIPTION
**Pull request by bboozzoo**
Populate modeenv with kernel command line during installation.

### CODE REVIEW

File: boot/makebootable.go [hunk 1]
```diff
@@ -356,6 +359,16 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
...
 			return fmt.Errorf("cannot install managed bootloader assets: %v", err)
 		}
+		// determine the expected command line
+		cmdline, err := ComposeCandidateCommandLine(model)
+		if err != nil {
+			return fmt.Errorf("cannot compose the candidate command line: %v", err)
+		}
+		modeenv.CurrentKernelCommandLines = bootCommandLines{cmdline}
+		// update the modeenv on disk
+		if err := modeenv.WriteTo(InstallHostWritableDir); err != nil {
```
Comment (pedronis) [inline boot/makebootable.go:369]: shouldn't we simply move the this out of the if and remove the previous WriteTo ?  writing modeenv twice, once partially is a bit strange to me

**Review by pedronis — COMMENTED** [review]
question

File: boot/makebootable.go (hunk 1 above)
Comment (bboozzoo) [reply boot/makebootable.go]: pushed

**Review by pedronis — APPROVED** [review]
thanks

**Review by mvo5 — APPROVED** [review]
Thank you!

**Review by anonymouse64 — APPROVED** [review]
LGTM, thanks for this

**Pull request merged**